src/util/config/types/SecurityConfiguration.ts

```diff
@@ -29,8 +29,7 @@ export class SecurityConfiguration {
 	// X-Forwarded-For for nginx/reverse proxies
 	// CF-Connecting-IP for cloudflare
 	forwadedFor: string | null = null;
-	ipdataApiKey: string | null =
-		"eca677b284b3bac29eb72f5e496aa9047f26543605efe99ff2ce35c9";
+	getipinfoEmail: string | null = null;
```

Comment:
I thought you said you expected the endpoint as input, not just an email? You should use the correspondence email in general instance config instead, I think